drivers/d3d12/rendering_device_driver_d3d12.h

```diff
@@ -970,7 +971,7 @@ class RenderingDeviceDriverD3D12 : public RenderingDeviceDriver {
 	virtual uint64_t get_total_memory_used() override final;
 	virtual uint64_t limit_get(Limit p_limit) override final;
 	virtual uint64_t api_trait_get(ApiTrait p_trait) override final;
-	virtual bool has_feature(Features p_feature) override final;
+	virtual bool has_feature(Features p_feature) const override final;
```

Comment:
@DarioSamo I couldn't get this compiled without making this a const, not sure if this somehow got overlooked due to me being on MSVC but I think this is part of  your restructure.

Reply:
Nope, the blame points to that line still being the same as the original commit by Pedro on both D3D12 and the original interface header. 

https://github.com/godotengine/godot/blob/8bad68df1a9504075450fbefe4f21fa8b2e9cf64/servers/rendering/rendering_device_driver.h#L765

It doesn't make any sense that you have to make it const. The original header has it as not const. Either way, both lines point to @RandomShaper so maybe you want to check with him what's up with that, but it doesn't sound right at all that you need to do this. What's the compiler error?

Reply:
I'm surprised this compiles without complaint on GCC (which is does in my testing). It's technically a different function with or without `const`, and I don't see the `const` version on any of the parents, and the `override` should fail if there isn't a parent with the method. Super weird! But compiles fine on GCC in any case.

Reply:
Hmm, well my compiler complains about it because it gets called by a const has_feature from the RD logic itself, and seeing a const function can't call a non const function, it doesn't compile. It's confusing...